include/deal.II/matrix_free/face_setup_internal.h

```diff
@@ -738,6 +739,11 @@ namespace internal
       const std::vector<std::pair<unsigned int, unsigned int>> &cell_levels,
       TaskInfo &                                                task_info)
     {
+      const auto &reference_cells = triangulation.get_reference_cells();
+      const bool  is_mixed_mesh   = reference_cells.size() > 1 ||
+                                 ((reference_cells[0].is_hyper_cube() ||
+                                   reference_cells[0].is_simplex()) == false);
```

Comment:
Does it make sense to move this into `Triangulation`?

Reply:
I think this is simple enough to keep it here, as long as it is not used more widely. But it's good to keep an eye on it.

Reply:
I believe that such a function could be useful in the future for other purposes, so I think it would be well suited for `Triangulation`. Also, we might forget that this functionality already exists here, and duplicate code in the future.

I am pro `Triangulation::is_mixed_mesh()`.

Reply:
https://github.com/dealii/dealii/pull/13044^^

Reply:
Yes, I just saw that :D
https://www.dealii.org/developer/doxygen/deal.II/classTriangulation.html#ab2cc397e18b3eaf0e8709abea714e17a